packages/server/src/database/knex/migrations/20200428185153_create-reveals.js

```diff
@@ -0,0 +1,30 @@
+
+export function up(knex) {
+  return knex.schema.hasTable('Reveals').then(function(sequelizeExists) {
+    if (sequelizeExists) {
+      return knex.schema.alterTable('Reveals', function (table) {
+        table.boolean('revealed').defaultTo(false).notNullable().alter()
```

Comment:
@facuspagnuolo I've added default value false here, seems appropriate. Also we may want to add some extra indexes here not sure how you query this stuff.